microsoft-edge/privacy-whitepaper/index.md

```diff
@@ -413,7 +413,7 @@ When you browse the web using **InPrivate**, the **Ask** and **Block** permissio
 
 History supports views to display images of your web navigations and enhance your history experience.  All history data is stored locally on the device, and is organized per Microsoft Edge profile.  Images are saved for the duration the history exists (90 days).
 
-If you want Microsoft Edge not to capture images for your history, select the **Turn off images** button in the history hub.  Or, go to `edge://settings/privacy`, scroll down to the **Services** section, and then turn off the toggle for **Save screenshots of site for History**.  All previously saved images will be erased.
+If you want Microsoft Edge not to capture images for your history, select the **Turn off images** button in the history hub.  Or, go to `edge://settings/privacy`,<!--todo: check, maybe replace by GUI nav path--> scroll down to the **Services** section, and then turn off the toggle for **Save screenshots of site for History**.  All previously saved images will be erased.
```

Comment:
this should be: **edge://settings/searchServices**

 **Settings and more** (**...**) > **Settings** > **Search and services**.

Reply:
```
Or, select **Settings and more** (![Settings and more icon](./index-images/settings-and-more-icon.png)) 
> **Settings** > **Search and services**, and then turn off the toggle 
for **Save screenshots of site for History**.
```